salt/modules/system.py

```diff
@@ -608,3 +609,54 @@ def get_computer_name():
         salt '*' network.get_hostname
     '''
     return __salt__['network.get_hostname']()
+
+
+NILRT_REBOOT_WITNESS_PATH = '/var/volatile/tmp/salt/reboot_witnessed'
+
+NILRT_FAMILY_VERIFY_CMD = 'sh -c \'lsb_release -d | grep -qi "NI Linux Real-Time"\''
```

Comment:
It has to be better than this...
```
$ lsb_release -d
-bash: lsb_release: command not found
```

Reply:
Done, I'm using the os family grain now.